lib/internal/process.js

```diff
@@ -283,6 +283,53 @@ function setupUncaughtExceptionCapture(exceptionHandlerState) {
   };
 }
 
+function setupCompareVersion() {
+  const {
+    majorVersion,
+    minorVersion,
+    patchVersion,
+  } = process.release;
+  const prereleaseTag = process.release.prereleaseTag.replace(/^-/, '');
```

Comment:
Isn't this `replace()` unnecessary since `substr(1, ..)` is already done in C++ land?